tests/sentry/issues/test_escalating.py

```diff
@@ -9,27 +9,23 @@
 from sentry.issues.escalating import (
     GroupsCountResponse,
     _start_and_end_dates,
-    get_group_daily_count,
+    get_group_hourly_count,
     is_escalating,
     query_groups_past_counts,
 )
 from sentry.issues.escalating_group_forecast import EscalatingGroupForecast
 from sentry.models import Group
 from sentry.models.group import GroupStatus
 from sentry.models.groupinbox import GroupInbox
-from sentry.testutils import TestCase
-from sentry.testutils.factories import Factories
+from sentry.testutils import SnubaTestCase, TestCase
```

Comment:
I have moved some of your test refactoring to #48247 so we don't have conflicting changes.